detect_secrets/audit/report.py

```diff
@@ -0,0 +1,137 @@
+import hashlib
+import codecs
+import json
+from enum import Enum
+
+from .io import print_message
+from ..core.plugins.util import Plugin, get_mapping_from_secret_type_to_class
+from ..core.scan import _get_lines_from_file, _scan_line
+from ..core.potential_secret import PotentialSecret
+from ..plugins.base import BasePlugin
+
+
+class SecretClass(Enum):
+    TRUE_POSITIVE = 1
+    FALSE_POSITIVE = 2
+    UNKNOWN = 3
+
+    def from_boolean(is_secret: bool) -> Enum:
+        if is_secret == None:
+            return SecretClass.UNKNOWN
+        elif is_secret:
+            return SecretClass.TRUE_POSITIVE
+        else:
+            return SecretClass.FALSE_POSITIVE
+
+    def to_string(self) -> str:
+        return self.name
+
+    def get_prioritary(self, secret_class: str) -> Enum:
+        try:
+            to_compare = SecretClass[secret_class]
+        except:
+            return self
+        if to_compare.value < self.value:
+            return secret_class
+        else:
+            return self
+
+
+class SecretClassToPrint(Enum):
+    REAL_SECRET = 1
+    FALSE_POSITIVE = 2
+
+    def from_class(secret_class: SecretClass) -> Enum:
+        if secret_class in [SecretClass.UNKNOWN, SecretClass.TRUE_POSITIVE]:
+            return SecretClassToPrint.REAL_SECRET
+        else:
+            return SecretClassToPrint.FALSE_POSITIVE        
+
+
+def print_report(
+    baseline_file: str,
+    class_to_print: SecretClassToPrint = None
+) -> None:
+    baseline = json.load(codecs.open(baseline_file, encoding='utf-8'))
+    details = get_secrets_details_from_baseline(baseline)
```

Comment:
v1 actually makes this a lot easier. This looks more like:

```python
for filename, secret in get_baseline_from_file(baseline_file):
    ...
```

Check out `detect_secrets.audit.analytics` for examples.

Reply:
Done!